x-pack/plugins/stack_connectors/public/connector_types/gen_ai/gen_ai.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import type { GenericValidationResult } from '@kbn/triggers-actions-ui-plugin/public/types';
+import { SUB_ACTION } from '../../../common/gen_ai/constants';
+import { GEN_AI_CONNECTOR_ID, GEN_AI_TITLE } from '../../../common/gen_ai/constants';
+import { GenerativeAiActionParams, GenerativeAiConnector } from './types';
+
+interface ValidationErrors {
+  subAction: string[];
+  body: string[];
+}
+export function getConnectorType(): GenerativeAiConnector {
+  return {
+    id: GEN_AI_CONNECTOR_ID,
+    iconClass: lazy(() => import('./logo')),
+    selectMessage: i18n.translate('xpack.stackConnectors.components.genAi.selectMessageText', {
+      defaultMessage: 'Send a request to Generative AI',
```

Comment:
Per https://en.wikipedia.org/wiki/Generative_artificial_intelligence this term seems to require another qualifier (e.g. "systems" or "models")
```suggestion
      defaultMessage: 'Send a request to generative AI systems.',
```